### ISSUE TITLE
Testing TAN for testing the report flow

### ISSUE THREAD

**Issue by corneliusroemer** [corona-warn-app/cwa-documentation#305]
A lot of valuable issues have been created since the general public has been able to use the app.

One part that is not reachable for the interested public to test is the "when positively diagnosed report" flow. It's something that we can only get to once positively tested. It is thus particularly badly tested and error prone.

Question: Is there a test/mock TAN like "0000-0000-0000" that can be entered to test the flow for crashes/UI/bugs? Of course, it should be visible that this is a "mock" report, but it would allow a much wider audience with a bigger assortment of devices to check whether the app functions correctly in this so very crucial use case.

**Comment by kbobrowski** [comment]
valid TAN is actually very easy to guess, I've guessed it multiple times, e.g. 222-222-2223 works

**Comment by tens0rfl0w** [comment]
> valid TAN is actually very easy to guess, I've guessed it multiple times, e.g. 222-222-2223 works

This is not a valid TAN, it just matches the TAN format, so it can get submitted to the verification server. Questioner asks for a TAN that is actually accepted by the server, so we can have a look at the screens, after submitting a valid TAN.

**Comment by kbobrowski** [comment]
@tens0rfl0w of course, it's just matching format, I was not precise enough. Expected behavior after sending this is that "TAN is invalid".

**Comment by tens0rfl0w** [comment]
@kbobrowski All good, just wanted to clarify so the question does not gets closed as „solved“😊

**Comment by SebastianWolf-SAP** [comment]
Dear @corneliusroemer,

thank you very much for your proposal. To answer your question, we don't have anything like that in place, we also don't have anything like that in the backlog. So we consider this a feature request.

To give the community a space for discussing new feature ideas or enhancements that might not be taken up immediately by the development teams, we created the cwa-wishlist repository.

This idea is one of these feature ideas or enhancements. Consequently, we will move this issue to this repository and allow the community and you to further refine it, discuss pros and cons, and evaluate alternatives.

The issue will not be closed in this repository to ensure long-term visibility. All issues in the cwa-wishlist repository will be discussed with our contracting entities. They will decide whether they will be implemented or pursued further. As soon as we have any updates, we will let you know about the details.

Best regards,
SW
Corona-Warn-App Open Source Team
